test/hotspot/gtest/nmt/test_vmatree.cpp

```diff
@@ -339,6 +365,18 @@ TEST_VM_F(NMTVMATreeTest, SummaryAccounting) {
   }
 }
 
+TEST_VM_F(NMTVMATreeTest, SummaryAccounting_dup) {
```

Comment:
Better name for the test, please avoid underscores in test names. I know, we have underscores in places in Hotspot, but it's a good rule for GTest naming.

Reply:
Done.